app/services/users.js

```diff
@@ -0,0 +1,3 @@
+const { User } = require('../models');
+
+exports.createUser = user => User.create(user);
```

Comment:
As I mentioned, add your own custom error for when the database fails, remember that we log the error that the database returns and throw a generic one.